code/datums/spells/lightning.dm

```diff
@@ -72,26 +70,18 @@
 	playsound(get_turf(usr), 'sound/magic/lightningbolt.ogg', 50, 1)
 	user.Beam(target,icon_state="lightning",icon='icons/effects/effects.dmi',time=5)
 
-	switch(energy)
-		if(1 to 25)
-			target.electrocute_act(10,"Lightning Bolt")
-			playsound(get_turf(target), 'sound/magic/LightningShock.ogg', 50, 1, -1)
-		if(25 to 75)
-			target.electrocute_act(25,"Lightning Bolt")
-			playsound(get_turf(target), 'sound/magic/LightningShock.ogg', 50, 1, -1)
-		if(75 to 100)
-			//CHAIN LIGHTNING
-			Bolt(user,target,energy,user)
+	var/energy = min(world.time - start_time,100)
+	Bolt(user,target,max(15,energy/2),5,user) //5 bounces for energy/2 burn
```

Comment:
But both of them are user and called at the same time, presumably therefore inputting the same value unless the variable is set to both at the same time?